pkg/start/start.go

```diff
@@ -132,15 +132,19 @@ func (o *Options) Run(ctx context.Context) error {
 	}
 	// check to see if techpreview should be on or off.  If we cannot read the featuregate for any reason, it is assumed
 	// to be off.  If this value changes, the binary will shutdown and expect the pod lifecycle to restart it.
-	includeTechPreview := false
+	startingFeatureSet := ""
 	gate, err := cb.ClientOrDie("feature-gate-getter").ConfigV1().FeatureGates().Get(ctx, "cluster", metav1.GetOptions{})
 	switch {
 	case apierrors.IsNotFound(err):
-		includeTechPreview = false // if we have no featuregates, then we aren't tech preview
+		// if we have no featuregates, then we assume the default featureset, which is "".
+		// This excludes everything that could possibly depend on a different feature set.
+		startingFeatureSet = ""
 	case err != nil:
 		klog.Warningf("Error getting featuregate value: %v", err)
+
 	default:
-		includeTechPreview = gate.Spec.FeatureSet == configv1.TechPreviewNoUpgrade
+		// otherwise, you're the default
```

Comment:
what is this comment about?  Just reminding us that we're in the `default:` case?